Assets/Scripts/Drone/Tutorial/Tutorial_Movement01_PlayCutScene.cs

```diff
@@ -0,0 +1,114 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using FiniteStateMachine;
+using UnityEngine.InputSystem;
+using Cinemachine;
+
+public class Tutorial_Movement01_PlayCutScene : MonoBehaviour
+{
+    [SerializeField]
+    private Drone _drone;
+    [SerializeField]
+    private DroneMovement _movement;
+
+
+    [Header("Tutorial Options")]
+    [SerializeField]
+    private bool _skipTutorial;
+    [SerializeField]
+    private float _droneDistanceToStopInFrontOfPlayer;
+    [Tooltip("This specifies how long of a delay between the end of the cutscene and when the drone flies onto the screen to help the player.")]
+    [SerializeField, Min(0)]
+    private float _droneInterceptDelay = 3f;
```

Comment:
It might be possible to use min duration on transitions to remove some of the time tracking code.

Reply:
I couldn't do that here, as the outer if statement means it wouldn't work.